classes/User.php

```diff
@@ -1303,6 +1326,9 @@ public function updateData() {
 
         $query .= "`battle_id` = '$this->battle_id',
 		`challenge` = '$this->challenge',
+		`last_movement` = '$this->last_movement',
+		`attack_id` = '$this->attack_id',
+		`attack_id_time` = '$this->attack_id_time',
```

Comment:
I don't see any of these three fields in the migration file, should there be another migration for them?

Also for `last_movement`, I'd like to call that `last_movement_ms` so we differentiate between fields storing time in seconds vs milliseconds